src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -97,6 +99,9 @@ class Settings extends Component {
       error: function(errorThrown) {
         console.log(errorThrown);
       },
+      complete: function(jqXHR, textStatus) {
+        location.reload();
+      },
```

Comment:
@Akshat-Jain Can we find a solution better than reloading the page every time we remove a device, looks good Otherwise.